.circleci/config.yml

```diff
@@ -1,59 +0,0 @@
-version: 2.1
-
-jobs:
-  build:
-    working_directory: ~/slovensko-digital/navody.slovensko.digital
-
-    docker:
-      - image: circleci/ruby:2.7.2-node-browsers
-        environment:
-          CC_TEST_REPORTER_ID: 17d259e00bbff5b020b7f1a6abb6736e0f65b9a517afdcbaeaa79e919730835f
-          PGHOST: localhost
-          RAILS_ENV: test
-
-      - image: circleci/postgres:12-alpine-ram
-        environment:
-          POSTGRES_USER: circleci
-          POSTGRES_HOST_AUTH_METHOD: trust
-
-    steps:
-      - checkout
-      - run: echo "deb http://apt.postgresql.org/pub/repos/apt stretch-pgdg main" | sudo tee -a /etc/apt/sources.list.d/pgdg.list
-      - run: wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
-      - run: sudo apt-get update
-      - run: sudo apt-get install postgresql-client-12
-      - run: curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./test-reporter
-      - run: chmod +x ./test-reporter
-      - restore_cache: { keys: ['{{ checksum "Gemfile.lock" }}'] }
-      - run: gem update --system
-      - run: gem install bundler
-      - run: bundle config set deployment true
-      - run: bundle config set path vendor/bundle
-      - run: bundle install
-      - save_cache: { key: '{{ checksum "Gemfile.lock" }}', paths: [vendor/bundle] }
-      - restore_cache: { keys: ['{{ checksum "yarn.lock" }}'] }
-      - run: yarn install
-      - save_cache: { key: '{{ checksum "yarn.lock" }}', paths: [~/.cache/yarn] }
-      - run: dockerize -wait tcp://localhost:5432 -timeout 60s
-      - run: bundle exec rake db:create db:structure:load --trace
-      - run: ./test-reporter before-build
-      - run: bundle exec rspec
-      - run: ./test-reporter after-build --coverage-input-type simplecov --exit-code $?
```

Comment:
@luciajanikova coverage nam nevypadol? a inak nie je na to nejaka oficialna action?

Reply:
Nasiel som toto https://gist.github.com/jcypret/608731eed983dd46a9e7447791c86cdb

Reply:
@jsuchal mozno skor nejaky action https://github.com/marketplace?query=code+climate&type=actions ale inak preco nam treba spustat ten `test-reporter`, nefunguje code climate aj bez toho?

Reply:
@pavolzbell toto je coverage report ktory musis do CC poslat, cize nie.